.travis.yml

```diff
@@ -6,6 +6,7 @@ stages:
 - linting
 - test
 - deploy
+dist: trusty
```

Comment:
still if found as per doc Travis using `Ubuntu Trusty 14.04 default` even other thought; what its breaking? `Xenial` should be comparable for both `py2` and `py3` so just confused.

Reply:
@digitronik It fails in installing pycurl, this issue is not in py3.7, but I have seen it on all the PRs opened today.